src/main/resources/public/error/404.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
This is more of a front-end feature. Absolutely 100% needed...but not here ;) From java already serve up a 404, and your web front-end will take that code and serve up a UI-themed 404 custom HTML site. :)